### PR TITLE
[Bug] When Complex datatype is provided in DICTIONARY_EXCLUDE, validated and throwing unsupported exception.

### PR DESCRIPTION
**Pull request by nareshpr**

### CODE REVIEW

**Comment by nareshpr** [comment]
retest this please

File: integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala [hunk 1]
```diff
@@ -556,6 +556,11 @@ class CarbonSqlParser()
             val errormsg = "DICTIONARY_EXCLUDE column: " + dictExcludeCol +
               " is no exist in table. Please check create table statement."
             throw new MalformedCarbonCommandException(errormsg)
+          } else if (isComplexDimDictionaryExclude(fields.filter(x =>
+              x.column.equalsIgnoreCase(dictExcludeCol))(0).dataType.get)) {
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala]: Instead use `find` like 
`fields.find(x => x.column.equalsIgnoreCase(dictExcludeCol)).get`

**Comment by ravipesala** [comment]
LGTM. Apart from one review comment.

**Comment by nareshpr** [comment]
Fixed review comment

**Comment by ravipesala** [comment]
LGTM

**Pull request merged**